viral/run_oasis.py

```
@@ -5,33 +5,27 @@
 
 HERE = Path(__file__).parent
 sys.path.append(str(HERE.parent))
 sys.path.append(str(HERE.parent.parent))
-sys.path.append(str(HERE.parent / "OASIS"))
 
-from OASIS.oasis.functions import deconvolve
+from oasis.functions import (
+    deconvolve,
+)
 
+from oasis.plotting import simpleaxis
```

Ruff (0.11.9)

`oasis.plotting.simpleaxis` imported but unused

Remove unused import: `oasis.plotting.simpleaxis`

(F401)